poc/README.md

```diff
@@ -8,16 +8,16 @@ generate test vectors.
 
 ## Installation
 
-This code requires [SageMath](https://www.sagemath.org/) version >= 9.6 to run. To install:
+This code requires Python 3.12 or later to run. To install:
 
 ```
-sage -pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@draft-irtf-cfrg-vdaf-11#subdirectory=poc
+pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@draft-irtf-cfrg-vdaf-11#subdirectory=poc
```

Comment:
We suggest elsewhere to invoke pip this way. Unless, is there a different affect you intend here?
```suggestion
python -m pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@draft-irtf-cfrg-vdaf-11#subdirectory=poc
```

Reply:
Good point, `python -m pip` just tends to be a bit more foolproof on systems with multiple Python installations, and inconsistent scripts in each.